ios/how-to-ios-setup-push-notifications.md

```diff
@@ -0,0 +1,247 @@
+# Setting Up iOS Push Notifications
+
+Push notifications for iOS apps are sent using Apple Push Notification service (APNs). Before you can send push notifications to iOS devices, you must create an app ID on the Apple Developer portal, and you must create the required certificates.
+
+This section describes how to use the Apple Developer portal to obtain iOS and APNs credentials. These credentials enable you to create an iOS project in AWS Mobile Hub and launch a sample app that can receive push notifications.
+
+You do not need an existing iOS app to complete the steps in this section. After you create an iOS project in Mobile Hub, you can download and launch a working sample app. Mobile Hub automatically provisions the AWS resources that your app requires.
```

Comment:
We shouldn't have these instructions as Mobile Hub isn't around any longer. The steps should use `amplify add notifications` like in this guide: https://docs.aws.amazon.com/aws-mobile/latest/developerguide/add-aws-mobile-push-notifications.html

Reply:
Done.